lib/node_modules/@stdlib/lapack/base/dlascl/lib/base.js

```diff
@@ -0,0 +1,558 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable max-len */
+
+'use strict';
+
+// MODULES //
+
+var isRowMajor = require( '@stdlib/ndarray/base/assert/is-row-major' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var dlamch = require( '@stdlib/lapack/base/dlamch' );
+var loopOrder = require( '@stdlib/ndarray/base/nullary-loop-interchange-order' );
+var max = require( '@stdlib/math/base/special/fast/max' );
+var min = require( '@stdlib/math/base/special/fast/min' );
+
+
+// VARIABLES //
+
+var smlnum = dlamch( 'safe minimum' );
+var bignum = 1.0 / smlnum;
+
+
+// FUNCTIONS //
+
+/**
+* Multiplies a real M by N general matrix `A` by a real scalar `mul`.
+*
+* @private
+* @param {NonNegativeInteger} M - number of rows in matrix `A`
+* @param {NonNegativeInteger} N - number of columns in matrix `A`
+* @param {Float64Array} A - input matrix
+* @param {integer} strideA1 - stride of the first dimension of `A`
+* @param {integer} strideA2 - stride of the second dimension of `A`
+* @param {NonNegativeInteger} offsetA - starting index for `A`
+* @param {number} mul - scalar multiplier
+* @returns {void}
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var A = new Float64Array( [ 1.0, 4.0, 2.0, 5.0, 3.0, 6.0 ] );
+*
+* scaleGeneral( 3, 2, A, 2, 1, 0, 2.0 );
+* // A => <Float64Array>[ 2.0, 8.0, 4.0, 10.0, 6.0, 12.0 ]
+*/
+function scaleGeneral( M, N, A, strideA1, strideA2, offsetA, mul ) {
+	var da1;
+	var da0;
+	var S1;
+	var S0;
+	var ia;
+	var i0;
+	var i1;
+	var o;
+
+	// Resolve the loop interchange order:
+	o = loopOrder( [ M, N ], [ strideA1, strideA2 ] );
+	S0 = o.sh[ 0 ];
+	S1 = o.sh[ 1 ];
+	da0 = o.sx[ 0 ];
+	da1 = o.sx[ 1 ] - ( S0 * o.sx[ 0 ] );
+
+	// Set the pointers to the first indexed elements in the respective matrices...
```

Comment:
```suggestion
	// Set the pointer to the first indexed element:
```

Applies here and anywhere else, as only one pointer.